src/Integration.TeamExplorer/Integration.TeamExplorer.csproj

```diff
@@ -53,6 +53,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\ConnectedMode\ConnectedMode.csproj" />
```

Comment:
The `TeamExplorer` now needs depends on types in `ConnectedMode` so it should reference it.